src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -76,6 +76,8 @@ public function map(Pagerfanta $pager): array
                 'language' => $contentInfo->mainLanguageCode,
                 'contributor' => $contributor,
                 'version' => $content->versionInfo->versionNo,
+                'type' => $content->getContentType()->getName(),
+                'content_type' => $content->getContentType(),
```

Comment:
What's the point of having these two? `content_type` is sufficient if this goes to the twig.

Reply:
It is @adamwojs part, but I think that it is my error made during rebasing.